foreman-proxy/foreman-proxy.service

```diff
@@ -8,6 +8,7 @@ Type=forking
 User=foreman-proxy
 ExecStart=/usr/share/foreman-proxy/bin/smart-proxy
 PIDFile=/run/foreman-proxy/foreman-proxy.pid
+TimeoutSec=infinity
```

Comment:
Okay. It was likely removed from documentation: https://www.freedesktop.org/software/systemd/man/systemd.service.html